doc/nrf/libraries/modem/nrf_modem_lib/nrf_modem_lib_lte_net_if.rst

```diff
@@ -0,0 +1,17 @@
+.. _nrf_modem_lib_lte_net_if:
+
+LTE Network Interface
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The LTE Network interface is intended to be integrated with and bound to a Zephyr network interface via the Connection Manager binding structure :ref:`conn_mgr_conn_api`.
+It can be managed via Zephyr's network management API and the Connection Manager Connectivity API using calls such as the :c:func:`net_if_down`, :c:func:`net_if_up`, and :c:func:`conn_mgr_if_connect` functions.
+
+.. note::
+
+   The nRF modem supports both IPv6 and IPv4. At any time, either IP families can be given by the network, independently of each other.
```

Comment:
We should not call it the `nRF modem`

Reply:
I would rephrase this entire part by saying just

```
The LTE network interface supports both IPv4 and IPv6.
The support is enabled automatically for whichever IP stack version is enabled in Zephyr.
```

Reply:
And this brings me to a question.. should we allow to disable IPv4 and use only IPv6?

Reply:
It's not a big change so I can add that.